packaging/systemd/flotta-agent.service

```diff
@@ -7,8 +7,12 @@ Before=yggdrasild.service
 [Service]
 Type=oneshot
 RemainAfterExit=yes
+ExecStart=bash -c "mkdir -p /var/home"
+ExecStart=bash -c "semanage fcontext -a -e /home /var/home"
+ExecStart=bash -c "restorecon -R -v /var/home"
```

Comment:
Can we document that those lines are just a workaround?

Reply:
will do.